dpbench/infrastructure/benchmark.py

```diff
@@ -54,9 +56,25 @@ def _setup_func(initialized_data, array_args, framework):
     return copied_args
 
 
+def _array_size(array: Any) -> int:
+    if isinstance(array, np.ndarray):
+        return array.size * array.itemsize
+    # for some reason runtime treat type as dpnp_array.dpnp_array
+    if isinstance(array, dpnp.dpnp_array.dpnp_array):
```

Comment:
does not `isinstance(array, dpnp.ndarray)` work?

Reply:
Unfortunately, no